src/app/models/Project.ts

```diff
@@ -86,3 +86,11 @@ export class Project implements ProjectInterface {
     };
   }
 }
+
+export const mockProject = new Project({
+  id: 1,
+  name: "name",
+  description: "description",
+  creatorId: 1,
+  siteIds: new Set([1])
+});
```

Comment:
Creating a mock project may be a bad practice for the unit tests which implement it

Reply:
Generally faking frameworks can help us generate better test data: https://github.com/marak/faker.js